src/pages/token-metadata/mint.md

```diff
@@ -251,17 +251,17 @@ create_cpi.invoke();
 {% /dialect %}
 {% /dialect-switcher %}
 
-## Minting Tokens
+## Minting Tokens [#](#minting-tokens)
 
-Once all on-chain accounts are created for our asset, we can mint tokens for it. If the asset is Non-Fungible we will simply mint its one and only token, otherwise we can mint as many tokens as we want. Note that a Non-Fungible asset is only valid once its unique token has been minted so it is a mandatory step for that Token Standard.
+Once all on-chain accounts are created for our NFT, we can mint tokens for it. If the NFT is Non-Fungible we will simply mint its one and only token, otherwise we can mint as many tokens as we want. Note that a Non-Fungible NFT is only valid once its unique token has been minted so it is a mandatory step for that Token Standard.
```

Comment:
Here's an example where calling them all NFTs confuses the meaning of the sentence.